### PR TITLE
describe fixture properties dialog

### PR DESCRIPTION
**Pull request by MargaritaLoseva**

### CODE REVIEW

File: documentation/user-interface/fixture-properties-dialog.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+---
+layout: docs
+title: Fixture Properties Dialog
+permalink: /documentation/user-interface/fixture-properties-dialog.html
+---
+# Fixture Properties Dialog
+
+The **Fixture Properties** dialog opens when you [create a new fixture](../guides/record-tests/README.md#create-fixtures) and allows you to specify its properties.
+
+The dialog also allows you to [modify](fixture-editor.md#modify-the-fixtures-properties) an existing fixture's properties.
```
Comment (VasilyStrelyaev) [inline documentation/user-interface/fixture-properties-dialog.md]: Maybe rephrase a bit:

```
The **Fixture Properties** dialog opens when you [create a new fixture](../guides/record-tests/README.md#create-fixtures) or [modify an existing one](fixture-editor.md#modify-the-fixtures-properties). Use it to specify or edit fixture's properties.
```

File: documentation/user-interface/fixture-properties-dialog.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+![Fixture Properties dialog](../../images/user-interface/dialogs/new-fixture-dialog.png)
+
+You can specify the following properties:
+
+* Fixture's name. When you enter a name, TestCafe automatically generates a file name and add it to the **File name** field.
+* File's name. You can use the generated name or specify your own.
+* Tested webpage's URL. TestCafe Studio supports HTTP and HTTPS protocols. You can also use the `file://` scheme to tests webpages in the local file system.
+* User credentials for webpages protected with HTTP [Basic](https://en.wikipedia.org/wiki/Basic_access_authentication) or [Windows (NTLM)](https://en.wikipedia.org/wiki/Integrated_Windows_Authentication) authentication. Expand the **HTTP Basic/NTLM authentication** section and specify a user name and password. For Windows authentication, TestCafe Studio also may need domain and workstation (PC) names. This information is automatically received from the machine on which TestCafe Studio is running.
```
Comment (helen-dikareva) [inline documentation/user-interface/fixture-properties-dialog.md]: I'm not sure about `domain and workstation (PC) names`, because in TestCafe open source we allow to set them if they are required. Do we always receive these parameters automatically? Or we should add them in this dialog? @churkin

File: documentation/user-interface/fixture-properties-dialog.md (hunk 1 above)
Comment (MargaritaLoseva) [reply documentation/user-interface/fixture-properties-dialog.md]: I thinks this phrase isn't exactly correct: "... dialog opens when you modify an existing one" because:
1. we open dialog TO MODIFY properties, not WHEN MODIFY
2. we don't modify the fixture, but ITS PROPERTIES.

Comment (VasilyStrelyaev) [reply documentation/user-interface/fixture-properties-dialog.md]: Okay, I just think that two paragraphs is too much for this information, that's why I tried to squeeze it in two shorter sentences.

File: documentation/user-interface/explorer-panel.md [hunk 3]
```diff
@@ -67,9 +61,9 @@ To run several tests from different fixtures and directories, select the tests u
 
 You can also use the context menu's ![Run test icon](../../images/user-interface/context-menu/run-tests.png) **Run Test** command to run tests.
 
-## Modify a Fixture
+## Modify a Fixture's Properties
```
Comment (churkin) [inline documentation/user-interface/explorer-panel.md]: Extra article

File: documentation/user-interface/fixture-properties-dialog.md (hunk 2 above)
Comment (churkin) [reply documentation/user-interface/fixture-properties-dialog.md]: We can get this info from the Windows machine, but not form the Mac or Linux. If client server requires this info then NTLM auth wont work on this platforms.

File: documentation/user-interface/explorer-panel.md (hunk 3 above)
Comment (MargaritaLoseva) [reply documentation/user-interface/explorer-panel.md]: no, this article is required:
- whose properties?  
- one fixture.

File: documentation/user-interface/fixture-properties-dialog.md (hunk 2 above)
Comment (churkin) [reply documentation/user-interface/fixture-properties-dialog.md]: https://github.com/DevExpress/testcafe-studio/issues/1715

Comment (MargaritaLoseva) [reply documentation/user-interface/fixture-properties-dialog.md]: I've removed info about domain and workstation names

**Comment by MargaritaLoseva** [comment]
@dirk-pieterse  Please, check the changes.
